Description: Adapt service files to libvirt-bin
 As long as we stick to the old naming for the init file.
Forwarded: no
Author: Stefan Bader <stefan.bader@canonical.com>

Index: libvirt-1.3.1/daemon/libvirtd.service.in
===================================================================
--- libvirt-1.3.1.orig/daemon/libvirtd.service.in	2016-01-28 13:27:48.000000000 +0100
+++ libvirt-1.3.1/daemon/libvirtd.service.in	2016-02-01 15:49:48.887561323 +0100
@@ -12,7 +12,7 @@ Documentation=http://libvirt.org
 
 [Service]
 Type=notify
-EnvironmentFile=-/etc/default/libvirtd
+EnvironmentFile=-/etc/default/libvirt-bin
 ExecStart=@sbindir@/libvirtd $libvirtd_opts
 ExecReload=/bin/kill -HUP $MAINPID
 KillMode=process
Index: libvirt-1.3.1/tools/libvirt-guests.service.in
===================================================================
--- libvirt-1.3.1.orig/tools/libvirt-guests.service.in	2016-01-28 13:27:48.000000000 +0100
+++ libvirt-1.3.1/tools/libvirt-guests.service.in	2016-02-01 15:50:19.551560851 +0100
@@ -1,6 +1,6 @@
 [Unit]
 Description=Suspend Active Libvirt Guests
-After=network.target libvirtd.service time-sync.target
+After=network.target libvirt-bin.service time-sync.target
 Documentation=man:libvirtd(8)
 Documentation=http://libvirt.org
 
